pkg/config/con.go

```
@@ -0,0 +1,894 @@
+		}
+
+		for _, configPath := range resolvedPaths {
+			found, configPath, err := cl.loadConfigFile(cl.atmosConfig, configPath, cl.viper)
+			if err != nil {
+				u.LogWarning(cl.atmosConfig, fmt.Sprintf("failed to merge configuration from '%s': %v", configPath, err))
+				continue
+			}
+			if found {
+				u.LogTrace(cl.atmosConfig, fmt.Sprintf("merge import paths: %v", resolvedPaths))
+			}
+			cl.deepMergeConfig()
```

golangci-lint (1.62.2)

Error return value of `cl.deepMergeConfig` is not checked

(errcheck)